paasta_tools/cli/cmds/push_to_registry.py

```diff
@@ -108,3 +136,43 @@ def paasta_push_to_registry(args):
             level='event',
         )
     return returncode
+
+
+def read_docker_registy_creds(registry_uri):
+    dockercfg_path = os.path.expanduser('~/.dockercfg')
```

Comment:
The old path is still valid: https://github.com/moby/moby/blob/master/cli/config/config.go#L18